release.nix

```diff
@@ -118,7 +118,7 @@ rec {
         [ makeWrapper autoconf automake libtool unzip nukeReferences pkgconfig sqlite libpqxx
           gitAndTools.topGit mercurial darcs subversion bazaar openssl bzip2 libxslt
           guile # optional, for Guile + Guix support
-          perlDeps perl
+          perlDeps perl nix
```

Comment:
I probably also ran into #319 
I think new versions of hydra will need nix anyway since s3binarycachestore was moved to nix: https://github.com/NixOS/nix/commit/d155d8015578c43953e4a9d1867e49c0b71534d7